msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Preapproval of RSC permissions
+description: In this article, learn more about to create, manage, and delete preapprovals of RSC enabled app in Microsoft Teams.
+ms.localizationpriority: medium
+author: MSFTRickyCastaneda
+ms.author: rcastaneda
+ms.topic: Conceptual
+ms.date: 08/29/2023
+---
+
+# Preapproval of RSC permissions
+
+Preapproval of Resource-Specific Consent (RSC) permission allows admins with detailed control over the RSC permissions an app can request during installation. App receives RSC permissions only at the time of installation. However, with preapproval policies, admins can predetermine the maximum permissions an app can request from the end user and obtain consent during installation. 
+
+To enable the preapproval of RSC permission policy for an app, admins must ensure the app can be installed in their organization. For more information, see [managing apps in Teams Admin Center](/microsoftteams/apps-in-teams). If the app can't be installed, you can't enable the preapproval of RSC permission for that specific app.
+
+Admins can create a detailed preapproval policy, based on the app ID, permissions, and the sensitivity of the accessed data. Preapproval of RSC permissions are designed for admins seeking to create advanced custom policies for their organization. 
+
+By default, the preapproval policies are managed by Microsoft, and we recommend that organizations must maintain `ManagedByMicrosoft` state. This state allows Microsoft's security team to deliver the best security for your organization.
+
+   > [!NOTE]
+      > Preapproval of RSC permissions is available only in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+      > Preapproval of RSC permissions and its operating procedures are subject to change.
+
```

Comment:
Needs to be fixed

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/15629902/6e9ac3a7-81ef-48d0-83ab-20918bd77b25)